### PR TITLE
Test/soraya

### PR DESCRIPTION
**Pull request by sorayaboza**
## Changes
_List Changes Introduced by this PR_
1. Replaced products.js with postgres.js.
2. Updated readme with instructions for running the app with PostgreSQL.

## Purpose
Introduce backend features to the app.

## Approach
Database allows for a larger set of items for our product list.

## Learning

Installing vs code with postgres: https://www.youtube.com/watch?v=ezjoDYs72GA

Connecting postgres database to the nextjs app:
 https://www.youtube.com/watch?v=EWo6dpy7tB8

Closes #000

### CODE REVIEW

File: .env
Comment (missile720) [inline .env:1]: This file should be in gitignore for security reasons. Looks like its just a sample so it should be fine

**Review by missile720 — CHANGES_REQUESTED** [review]
Looks good just add the .env file to .gitignore

**Review by jm2601 — APPROVED** [review]
Looks good. I have some trouble getting the DB to work because Mac issues however, these changes also fixed issue #19 .

**Review by missile720 — APPROVED** [review]
Looks good!

**Pull request merged**